src/test/java/org/fundacionjala/coding/Franco/EANValidationTest.java

```diff
@@ -0,0 +1,43 @@
+package org.fundacionjala.coding.Franco;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by Administrator on 8/29/2017.
+ */
+public class EANValidationTest {
```

Comment:
to increase the code coverage you should test when the checksum is 0.
for instance.
```java
 assertTrue(eanValidation.checksumValidation("9783827317100"));
```